build-utils/interpolate-all-results.js

```diff
@@ -0,0 +1,49 @@
+var parseEnvsVersions = require("./parse-envs-versions");
+
+module.exports = function interoplateAllResults(tests, envs) {
+  var envsTree = buildEnvsTree(envs);
+
+  tests.forEach(function (t) {
+    // Calculate the result totals for tests which consist solely of subtests.
+    if ("subtests" in t) {
```

Comment:
should this use `Object.prototype.hasOwnProperty.call(t, 'subtests')` rather than `in`?

Reply:
Yes!
I used `in` because it's used in other parts of the codebase, but I'm happy to remove it.